src/rest-client-v5.ts

```diff
@@ -1729,4 +1896,80 @@ export class RestClientV5 extends BaseRestClient {
   > {
     return this.getPrivate('/v5/ins-loan/ltv-convert');
   }
+
+  /**
+   * Bind or unbind UID for the institutional loan product.
+   *
+   * INFO
+   * Risk unit designated UID cannot be unbound
+   * This endpoint can only be called by uids in the risk unit list
+   * The UID must be upgraded to UTA Pro if you try to bind it.
+   * When the API is operated through the API Key of any UID in the risk unit, the UID is bound or unbound in the risk unit.
+   */
+  bindOrUnbindUID(params: { uid: string; operate: '0' | '1' }): Promise<
+    APIResponseV3WithTime<{
+      uid: string;
+      operate: '0' | '1';
+    }>
+  > {
+    return this.postPrivate('/v5/ins-loan/association-uid', params);
+  }
+
+  /**
+   *
+   ****** Broker
+   *
+   */
+
+  /**
+   * Get Exchange Broker Earning.
+   *
+   * INFO
+   * Use exchange broker master account to query
+   * The data can support up to past 1 months until T-1. To extract data from over a month ago, please contact your Relationship Manager
+   * begin & end are either entered at the same time or not entered, and latest 7 days data are returned by default
+   * API rate limit: 10 req / sec
+   */
+  getExchangeBrokerEarning(
+    params?: GetExchangeBrokerEarningParamsV5,
+  ): Promise<APIResponseV3WithTime<ExchangeBrokerEarningResultV5>> {
+    return this.getPrivate('/v5/broker/earnings-info', params);
+  }
+
+  /**
+   * Get Exchange Broker Account Info.
+   *
+   * INFO
+   * Use exchange broker master account to query
+   * API rate limit: 10 req / sec
+   */
+  getExchangeBrokerAccountInfo(): Promise<
+    APIResponseV3WithTime<ExchangeBrokerAccountInfoV5>
+  > {
+    return this.getPrivate('/v5/broker/account-info');
+  }
+
+  /**
+   * Get Sub Account Deposit Records.
+   *
+   * Exchange broker can query subaccount's deposit records by main UID's API key without specifying uid.
+   *
+   * API rate limit: 300 req / min
+   *
+   * TIP
+   * endTime - startTime should be less than 30 days. Queries for the last 30 days worth of records by default.
+   */
+  getBrokerSubAccountDepositRecords(
+    params?: getBrokerSubAccountDepositRecords,
```

Comment:
```suggestion
    params?: GetBrokerSubAccountDepositsV5,
```

(make sure to fix the imports and test the build after taking these changes)